src/main/java/Duke.java

```diff
@@ -1,10 +1,168 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+// I am using the Google Java Style Guide https://google.github.io/styleguide/javaguide.html
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+  static ArrayList<Task> enteredCommands=new ArrayList<>();
+
+  public static void greet() {
+    System.out.printf("%s%n", "    ____________________________________________________________");
+    System.out.printf("%s%n", "     Hello! I'm CHAT-MAN");
+    System.out.printf("%s%n", "     What can I do for you?");
+  }
+
+  public static void exit() {
+    System.out.printf("%s%n","    ____________________________________________________________");
+    System.out.printf("%s%n","     Bye. Hope to see you again soon!");
+    System.out.printf("%s%n","    ____________________________________________________________");
+  }
+
+  public static void replyAddedTask(){
+    Task addedTask = enteredCommands.get(enteredCommands.size() - 1);
+    int size=enteredCommands.size();
+    System.out.printf("     Got it. I've added this task:%n      %s%n     Now you have %d tasks in the list.%n",
+        addedTask.toString(),size);
+  }
+
+  public static void addTask(String userCommand) {
+
+    System.out.printf("%s%n","    ____________________________________________________________");
+    String taskType=userCommand.split(" ")[0].toUpperCase();
+
+    switch (taskType) {
+      default:
+        break;
+
+      case "TODO":
+        String toDoDesc=userCommand.replaceAll("(?i)TODO","");
+        enteredCommands.add(new Todo(toDoDesc));
+        replyAddedTask();
+        break;
+
+      case "DEADLINE":
+        String[] deadLine=userCommand.split("/");
```

Comment:
```suggestion
        String[] deadLine = userCommand.split("/");
```
You might want to consider adding spaces before and after the "=" sign for better readability.